addOns/network/network.gradle.kts

```diff
@@ -44,11 +44,26 @@ crowdin {
     }
 }
 
+spotless {
+    java {
+        target(fileTree(projectDir) {
+            include("src/**/*.java")
+            exclude("src/main/java/org/apache/hc/client5/**/Zap*.java")
+        })
+    }
+}
+
 dependencies {
     val nettyVersion = "4.1.73.Final"
     implementation("io.netty:netty-codec:$nettyVersion")
     implementation("io.netty:netty-handler:$nettyVersion")
 
+    implementation("org.apache.httpcomponents.client5:httpclient5:5.2-beta1")
```

Comment:
I'm guessing/assuming this is the latest. Is there any special steps to be documented for upgrading? (Same for netty I guess).

Reply:
It is the latest of 5.2. It shouldn't require anything special (assuming there are no breaking changes between versions).

Reply:
Great!